linux/build.gradle

```diff
@@ -74,6 +82,22 @@ def getJdkArchitecture() {
         Architecture.X64
 }
 
+def getVendor() {
+    return hasProperty("VENDOR") ? VENDOR : "AdoptOpenJDK"
+}
+
+def getPackageName() {
+    return hasProperty("VENDOR") ? "openjdk" : "adoptopenjdk"
```

Comment:
I have doubts that every third party wants to call their package openjdk. I'd introduce a separate property `PACKAGE_NAME`.